### DISCUSSION TITLE
Help with [ERROR] [engine.core.Statistics]

### DISCUSSION THREAD

**Discussion by oduvaldo-vick** [sepinf-inc/IPED#2250] [Q&A] [answered]
I extracted a NVMe using Encase, then tried to process the E01 with IPED and I am receiving the error of the image below:

![image](https://github.com/sepinf-inc/IPED/assets/7217001/d2121118-3e5f-4e77-a3d0-a5a5c3d47f7c)

I have no idea what it means, I tried again and the numbers changed as you can see. Can someone clarify it to me?

[EDIT: Removed Azure Key]

**Comment by wladimirleite** [comment]
Hello @oduvaldo-vick!

This message means that there were a lot of I/O (reading) errors while processing the evidence.
Usually this is a bad sign, as many items were not processed correctly (they could be read).

Possible causes I can think of are hardware failure in the device in which the E01 is stored, network errors (in case the evidence in a network mapped volume), or possibly a corrupted E01 (not sure about this last one).
Is G: a local volume? If that is the case, try to copy the image to a local disk and process it again.
If not, try to use another local disk.

I would also try to open the E01 file in another tool (e.g. FTK Imager) and access some files, just to check if the image is not corrupted.

**Reply by lfcnassif** [reply, marked as answer]
Another possible cause is a corrupted File System in the E01 image, it happens when the suspect computer is shutdown abruptly and the file system is not repaired. But, in this situation, the number of IO read errors should repeat between different IPED runs.

Another cause might be your anti-virus blocking the reads from the E01 image or writes to the temp folder, but in this case usually the number of IO read errors is just a few, not that much.

There is a built-in filter on the UI top left corner which you can enable to see all those files with IO read errors, maybe that could help, if they don't open in the Hex viewer, the corrupted File System issue would be a stronger hypothesis.

**Reply by lfcnassif** [reply]
PS: you can try to open the same files listed in the "Read Error" filter I pointed above in FTKImager as @wladimirleite suggested.

**Reply by gfd2020** [reply]
Just a suggestion based on @wladimirleite tip.
Open the image in ftk imager, then click on the "Verify Drive/Image" option. 
At the end of processing, "Bad Block List" will appear.
Just to make sure the E01 is intact.

![image](https://github.com/sepinf-inc/IPED/assets/59742865/1833d62e-f625-4bed-b223-0bc64871fd7c)
